.github/workflows/clang-format.yml

```diff
@@ -0,0 +1,19 @@
+name: clang-format
+on: push
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: Ana06/get-changed-files@v2.0.0
+        id: gcf
+        with:
+          filter: |
+            **/*.cpp
+            **/*.hpp
+            **/*.cc
+            **/*.c
+            **/*.h
+      - uses: aMahanna/clang-format-checker@v1.2.2
```

Comment:
This will be changed to `arangodb/clang-format-action@v1.0.0` once we publish https://github.com/arangodb/clang-format-action

Reply:
Note: CI is failing (https://github.com/arangodb/arangodb/runs/4576063938?check_suite_focus=true) because the ArangoDB Clang-Format Action is yet to be published (see https://github.com/arangodb/clang-format-action)